src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,154 @@
-import { PeopleFilters } from './PeopleFilters';
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
 import { Loader } from './Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
 import { PeopleTable } from './PeopleTable';
+import { PeopleFilters } from './PeopleFilters';
+
+export enum SortingOptions {
+  name = 'name',
+  nameRev = 'nameRev',
+  sex = 'sex',
+  sexRev = 'sexRev',
+  born = 'born',
+  bornRev = 'bornRev',
+  died = 'died',
+  diedRev = 'diedRev',
```

Comment:
you have to sort only by name, sex, born and died keys, other cases depends from `order `
![Uploading image.png…]()